api/src/com/cloud/network/vpc/VpcProvisioningService.java

```diff
@@ -26,13 +25,11 @@
 
     public VpcOffering getVpcOffering(long vpcOfferingId);
 
-    public VpcOffering createVpcOffering(String name, String displayText, List<String> supportedServices,
-                                         Map<String, List<String>> serviceProviders,
-                                         Map serviceCapabilitystList,
-                                         Long serviceOfferingId);
+    public VpcOffering createVpcOffering(String name, String displayText, List<String> supportedServices, Map<String, List<String>> serviceProviders, Map serviceCapabilitystList,
+            Long serviceOfferingId);
```

Comment:
This appears to be a spurious formatting change that violates our coding standards.